client/packages/system/src/Item/Components/ColdStorageTypeInput/ColdStorageTypeInput.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Autocomplete, useBufferState } from '@openmsupply-client/common';
+import { useColdStorageTypes } from '../../api/hooks/useColdStorageTypes';
+import { ColdStorageTypeFragment } from '../../api';
+
+export interface ColdStorageTypeInputProps {
+  onChange: (name: ColdStorageTypeFragment) => void;
+  onInputChange?: (
+    event: React.SyntheticEvent,
+    value: string,
+    reason: string
+  ) => void;
+  width?: number;
+  value: ColdStorageTypeFragment | null;
+  disabled?: boolean;
+  clearable?: boolean;
+}
+
+export const ColdStorageTypeInput = ({
```

Comment:
Based this off ManufacturerInput, so it would look consistent, assume that would be ok for other areas too?